toolboxes/electric/quarter-turn/2d.json

```diff
@@ -0,0 +1,45 @@
+// -*- mode: javascript -*-
+{
+    "Name":"cvg2d",
+    "ShortName":"cvg2d",
+    "Models":
+    {
+        "v_expr":"2*0.25/Pi*atan2(x,y):x:y",
+        "c_expr":"{-2*0.25/Pi*y/(x*x+y*y),2*0.25/Pi*x/(x*x+y*y)}:x:y:z"
+    },
+    "Materials":
+    {
+        "omega":
+        {
+            "sigma":"4.8e7"
+        }
+    },
+    "BoundaryConditions":
+    {
+        "electric-potential":
+        {
+            "Dirichlet":
+            {
+                "V0":
+                {
+                    "expr":"0" // V_in
+                },
+                "V1":
+                {
+                    "expr":"0.25" // V_out
+                }
+            }
+        }
+    },
+    "PostProcess":
+    {
+        "use-model-name":1,
+        "electric":
+        {
+            "Exports":
+            {
+                "fields":["electric.electric-potential","electric.electric-field","electric.pid"]
```

Comment:
no electric prefix

Reply:
@vincentchabannes I've made the changes you request.
I still need to run the tests to  be 100% sure of the changes.